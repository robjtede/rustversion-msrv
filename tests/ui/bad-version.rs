#[rustversion_msrv::stable(nightly)]
struct S;

#[rustversion_msrv::any(stable(nightly))]
struct S;

fn main() {}
