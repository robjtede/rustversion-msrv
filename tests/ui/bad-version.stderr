error: expected rustc release number, like 1.31
 --> tests/ui/bad-version.rs:1:28
  |
1 | #[rustversion_msrv::stable(nightly)]
  |                            ^^^^^^^

error: expected rustc release number, like 1.31
 --> tests/ui/bad-version.rs:4:32
  |
4 | #[rustversion_msrv::any(stable(nightly))]
  |                                ^^^^^^^
